platform.mk

```diff
@@ -243,23 +243,21 @@ PRODUCT_PACKAGES += \
     hwcomposer.sm8450 \
     memtrack.default
 
-# Keymaster 4 passthrough service init file
+# KeyMint passthrough service init file
 # (executable is on odm)
 PRODUCT_PACKAGES += \
     android.hardware.security.keymint-V1-ndk_platform.vendor \
     libkeymaster_messages.vendor \
-    android.hardware.keymaster@4.1-service-qti.rc \
     android.hardware.keymaster@4.1.vendor \
```

Comment:
Same here, who still links the HIDL helper library if no-one exposes the interface?